README.md

```diff
@@ -101,5 +101,11 @@ We welcome contributions to the project. Detailed information on our ways of wor
 
 ## FAQ
 
-* What versions of Java are supported? Java 8 with plans to upgrade to Java 11+ soon.
+What versions of Java are supported? Java 8 with plans to upgrade to Java 11+ soon.
```

Comment:
To be precise, all development so far is JDK8, we have tested that everything builds and the examples work on JDK10. There are no Java architectural changes that should prevent JDK 11/12 from working. JDK 9 broke a lot of things when they removed JAXB and SAX XML processing from the JDK. I don't see any issues for JDK 12.

Reply:
I think for now I am going to leave the FAQ guidance as Java 8 is the only supported version, just because I don't think we have tested with Java 11/12 enough to be confident it will work for people